presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcessFactory.java

```diff
@@ -48,6 +49,8 @@
     private final TaskManagerConfig taskManagerConfig;
     private final WorkerProperty<?, ?, ?> workerProperty;
 
+    private static NativeExecutionProcess process;
```

Comment:
Are we relying on a static variable in a single JVM for cross task reuse of process?